script/options.js

```diff
@@ -57,7 +58,11 @@ function applyEnvironmentOverrides(options) {
     case environments.PRODUCTION:
       if (options['no-sanity-check-node-env'] === false) {
         if (env !== 'prod') {
-          throw new Error(`buildtype ${options.buildtype} expects NODE_ENV to be production, not '${process.env.NODE_ENV}'`);
+          throw new Error(
+            `buildtype ${
+              options.buildtype
+            } expects NODE_ENV to be production, not '${process.env.NODE_ENV}'`,
```

Comment:
Is there any way to disable whatever it is that does this? It seems...wrong to split strings like this.

Reply:
I'll look into this. It might be possible if we change the parser? But actually since we are using ESLint to parse, rather than standalone Prettier, that probably isn't a real option.

I'll dig a bit and see what I can find but I don't recall this being one of the Prettier options.

Reply:
@cvalarida [You're not the only one who feels this way](https://github.com/prettier/prettier/issues/3368). But there's not prettier config setting to disable it. It won't break lines at [*all* expressions](https://github.com/prettier/prettier/issues/3368#issuecomment-379310998), though.

Reply:
There's always `// eslint-disable-next-line prettier/prettier` (doesn't seem to be a way to disable just the `"printWidth"` rule.

Reply:
Small price to pay, I suppose.